environments/production/aws-resources.yml

```diff
@@ -34,6 +34,7 @@ pgshard5-production: pgshard5-production.cl9dmuo3ok4h.us-east-1.rds.amazonaws.co
 pgsynclog0-production: pgsynclog0-production.cl9dmuo3ok4h.us-east-1.rds.amazonaws.com
 pgucr0-production: pgucr0-production.cl9dmuo3ok4h.us-east-1.rds.amazonaws.com
 pillow0-production: 10.202.12.170
```

Comment:
should pillow0 be removed?

Reply:
We're generally okay with leaving this cache around for a bit, especially during the switchover while the old machine may still be relevant. But yeah next time someone runs `aws-fill-inventory` without `--cached` this'll get updated.